sql/hive/src/test/resources/regression-test-SPARK-8489/Main.scala

```diff
@@ -34,7 +34,7 @@ object Main {
     // scalastyle:off println
     println("Running regression test for SPARK-8489.")
     val sc = new SparkContext("local", "testing")
-    val sparkSession = SparkSession.withHiveSupport(sc)
+    val sparkSession = SparkSession.builder.enableHiveSupport().getOrCreate()
```

Comment:
Not sure I got you ? do you mean something like `withSparkContext(sc)` instead of `getOrCreate()`